### PR TITLE
Implement ANSSI requirement R69 for RHEL

### PR DESCRIPTION
**Pull request by vojtapolasek**
#### Description:

- add two similar rules which check for presence of obsolete services in /etc/nsswitch.conf
- each rule has only OVAL check, as remediation should be done manually
- services are NIS and Hesiod
- the wording of the requirement is updated

#### Rationale:

- ANSSI 2.0 alignment

### CODE REVIEW

**Comment by Mab879** [comment]
The failure on building the HTML seems suspicious, can you please investigate?

**Comment by jan-cerny** [comment]
The utils/gen_rendered_policies_index.py doesn't support jinja macros in control files, you need to change that because jinja macros should be allowed in control files.

**Comment by vojtapolasek** [comment]
The error while generating HTML pages is caused by the script gen_rendered_policies_index because it can't handle Jinja. Investigating...

**Comment by vojtapolasek** [comment]
I propose fix here: https://github.com/ComplianceAsCode/content/pull/11667

**Comment by Mab879** [comment]
I merged #11667 so you should be able rebase this PR to resolve the CI issues.

File: linux_os/guide/services/obsolete/nis/no_nis_in_nsswitch/rule.yml [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+identifiers:
+    cce@rhel7: CCE-86074-2
+    cce@rhel8: CCE-86076-7
+
+ocil_clause: "a nis database is configured in nsswitch.conf"
+
+ocil: |-
+    Run the following command:
+    <pre>grep '^\w+\s+(\w+\s+)*nis($|\s+.*$)' /etc/nsswitch.conf</pre>
+    If a line is returned and it    contains the word <tt>nis</tt> in the list
```
Comment (Mab879) [inline linux_os/guide/services/obsolete/nis/no_nis_in_nsswitch/rule.yml:27]:
```suggestion
    If a line is returned and it contains the word <tt>nis</tt> in the list
```

File: linux_os/guide/services/obsolete/no_hesiod_in_nsswitch/rule.yml [hunk 2]
```diff
@@ -0,0 +1,35 @@
+documentation_complete: true
+
+title: "Name Service Switch does not use Hesiod"
+
+description: |-
+    Each call to a function which retrieves data from a system database like the
+    password or group database is handled by the Name Service Switch
+    implementation in the GNU C library.  The various services provided are
```
Comment (Mab879) [inline linux_os/guide/services/obsolete/no_hesiod_in_nsswitch/rule.yml:8]:
```suggestion
    implementation in the GNU C library. The various services provided are
```

File: linux_os/guide/services/obsolete/no_hesiod_in_nsswitch/rule.yml [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+
+identifiers:
+    cce@rhel7: CCE-86078-3
+
+ocil_clause: "a hesiod service is configured in nsswitch.conf"
+
+ocil: |-
+    Run the following command:
+    <pre>grep '^\w+\s+(\w+\s+)*hesiod($|\s+.*$)' /etc/nsswitch.conf</pre>
+    If a line is returned and it    contains the word <tt>hesiod</tt> in the list
```
Comment (Mab879) [inline linux_os/guide/services/obsolete/no_hesiod_in_nsswitch/rule.yml:28]:
```suggestion
    If a line is returned and it contains the word <tt>hesiod</tt> in the list
```

**Review by Mab879 — CHANGES_REQUESTED** [review]
Just a few issues. 

Thanks for the PR!

**Comment by vojtapolasek** [comment]
Thank you for suggestions, applied.

**Review by Mab879 — APPROVED** [review]

**Pull request merged**